hugo/layouts/partials/article_modules/article_pre_article.html

```diff
@@ -7,7 +7,7 @@
   <div class="social-container">
     <a class="fa fa-facebook fa-2x social-icon fb-icon" href="http://www.facebook.com/sharer/sharer.php?u={{ .Permalink | safeURL }}&title={{ $shareTitle }}&picture={{ .Site.Params.shine_logo }}&description={{ .Site.Params.description }}" target="_blank"></a>
     <a class="fa fa-twitter fa-2x social-icon twitter-icon" href="http://twitter.com/share?via=ShineText&url={{ .Permalink | safeURL }}&text={{ .Params.title }}" target="_blank"></a>
-    <a class="fa fa-pinterest fa-2x social-icon pintrest-icon" data-pin-do="buttonPin" data-pin-tall="true" data-pin-round="true" data-pin-save="false" href="https://www.pinterest.com/pin/create/button/?url={{ .Permalink | safeURL }}&media={{ .Site.Params.shine_logo }}&description={{ .Site.Params.description }}" target="_blank"></a>
+    <a class="fa fa-pinterest fa-2x social-icon pinterest-icon" data-pin-do="buttonPin" data-pin-tall="true" data-pin-round="true" data-pin-save="false" href="https://www.pinterest.com/pin/create/button/?url={{ .Permalink | safeURL }}&media={{ .Site.Params.shine_logo }}&description={{ .Site.Params.description }}" target="_blank"></a>
```

Comment:
Instead of the shine logo, should we use the article image?

Reply:
`.Params.image`

Reply:
Hey @kaladin9017 I actually think that the media attribute is not being used because I'm using the data-pin-do="buttonBookmark" attribute, which allows you to pin any image on the page, including the featured one.

I'm going to remove the media attribute from the Pinterest link. For the Facebook link, should I replace picture={{ .Site.Params.shine_logo }} with picture={{ .Site.Params.bigimg }} ?

Reply:
From our IRL convo: no need to change the picture/media attributes. FB's debugger is autopulling the featured image at the top of the article. Pinterest now allows you to pull any image on the page. The logo is a fallback incase either of those don't exist.